### PR TITLE
users/config: Update config article for current config format (v13)

### PR DESCRIPTION
**Pull request by calmh**

### CODE REVIEW

File: users/config.rst [hunk 1]
```diff
@@ -49,8 +49,8 @@ The following shows the default configuration file:
 
 .. code-block:: xml
 
-    <configuration version="12">
-        <folder id="xsdasdSasd" label="default-sync" path="/Users/jb/Sync/" type="readwrite" rescanIntervalS="60" ignorePerms="false" autoNormalize="true">
+    <configuration version="13">
+        <folder id="zj2AA-q55a7" label="Default Folder (zj2AA-q55a7)" path="/Users/jb/Sync/" type="readwrite" rescanIntervalS="60" ignorePerms="false" autoNormalize="true" ro="false">
```
Comment (AudriusButkevicius) [inline users/config.rst]: Ro=false?

File: users/config.rst [hunk 2]
```diff
@@ -98,13 +99,14 @@ The following shows the default configuration file:
...
             <keepTemporariesH>24</keepTemporariesH>
-            <cacheIgnoredFiles>true</cacheIgnoredFiles>
+            <cacheIgnoredFiles>false</cacheIgnoredFiles>
             <progressUpdateIntervalS>5</progressUpdateIntervalS>
             <symlinksEnabled>true</symlinksEnabled>
             <limitBandwidthInLan>false</limitBandwidthInLan>
-            <databaseBlockCacheMiB>0</databaseBlockCacheMiB>
             <minHomeDiskFreePct>1</minHomeDiskFreePct>
             <releasesURL>https://api.github.com/repos/syncthing/syncthing/releases?per_page=30</releasesURL>
+            <overwriteNames>false</overwriteNames>
```
Comment (AudriusButkevicius) [inline users/config.rst]: We should rename this while we can, it's confusing as fuck.

File: users/config.rst (hunk 1 above)
Comment (calmh) [inline users/config.rst]: That's what we write. Missing omitempty I guess

File: users/config.rst (hunk 2 above)
Comment (calmh) [inline users/config.rst]: Bit late in the day for that. Maybe we should delay the v0.13 release for another week and take some time to figure this and the proper solution to the other PR out

Comment (AudriusButkevicius) [inline users/config.rst]: I think this is so low risk that we are fine to do it. Just notify people in the beta page.

Comment (AudriusButkevicius) [inline users/config.rst]: I bet my socks no external code handles this setting yet.

Comment (calmh) [inline users/config.rst]: What do you want to call it? `overwriteRemoteDeviceNamesOnConnect`?

Comment (AudriusButkevicius) [inline users/config.rst]: Yes

**Comment by calmh** [comment]
Updated for https://github.com/syncthing/syncthing/pull/3082

File: users/config.rst [hunk 3]
```diff
@@ -397,22 +418,28 @@ Options Element
...
 
 The ``options`` element contains all other global configuration options.
 
 listenAddress
     The listen address for incoming sync connections. See the ``address``
     element under the `GUI Element`_ for allowed syntax, with the addition
-    that the address must have a protocol scheme prefix. Currently ``tcp://``
-    is the only supported protocol scheme.
+    that the address must have a protocol scheme prefix. Currently
+    ``tcp://`` is the only supported protocol scheme. The default value of
```
Comment (AudriusButkevicius) [inline users/config.rst]: This is somewhat a lie, we have tcp4/6

Comment (calmh) [inline users/config.rst]: Indeed. I'll fix

**Comment by rumpelsepp** [comment]
We have v14 now: https://github.com/syncthing/syncthing/commit/5d2414dfa92cdcbcb55acb8b5d238482af964ec8

**Comment by calmh** [comment]
Fixed, extracted listen addresses for sync protocol into a separate section as it's not really a shared syntax with the listen address for the GUI. Needs a little bit more love, but I don't have the time right now :)

**Comment by AudriusButkevicius** [comment]
@st-review merge

**Comment by st-review** [comment]
:ok_hand: Merged as 0bef7754bb08317dccd78b27635b9b219aab618c. Thanks, @calmh!

**Comment by AudriusButkevicius** [comment]
Missing info about default tho

**Pull request closed without merging**